SimPEG/potential_fields/magnetics/simulation.py

```diff
@@ -36,7 +56,9 @@ def __init__(
         chiMap=None,
         model_type="scalar",
         is_amplitude_data=False,
-        **kwargs
+        engine="geoana",
+        choclo_parallel=True,
```

Comment:
I think these could get moved to the `Base` class correct? (as `numba_parallel` to match the gravity simulation)

Reply:
Yes! We should move it to the base class. I just added it to the todo list in the PR description. I will also try to move the sanity check for those parameters to the base class so we have only one of them.